build_script.sh

```diff
@@ -4,6 +4,12 @@ set -ev
 files=`git diff --name-only HEAD..$TRAVIS_BRANCH`
 if [[ $files = *"bigbluebutton-html5"* ]]; then
   cd bigbluebutton-html5
+  git clone --single-branch -b update-html5 https://github.com/bigbluebutton/docker.git
+  cp -r docker/mod .
+  cp -r docker/Dockerfile Dockerfile.test
+  cp -r docker/restart.sh .
```

Comment:
`cp -r` could be replaced by `cp` for the single files afaik
or even
`git clone --single-branch -b update-html5 https://github.com/bigbluebutton/docker.git`
`cp -r docker/{mod, restart.sh, setup.sh, dupervisord.conf} .`
`cp docker/Dockerfile Dockerfile.test`